install.sh

```diff
@@ -420,6 +336,69 @@ fi
 # The following function exits script if an unsupported distro is detected
 supported_distro
 
+function display_help()
+{
+cat <<EOF
+
+  hipBLAS library build & installation helper script.
```

Comment:
Okay but this is still duplicating rmake.py help right?   I think I only left in the install handled options in rocBLAS

Reply:
Removed any unneeded parsing in install.sh in latest commit.